### PR TITLE
DOCS-11772 update non-inclusive terms

### PR DESCRIPTION
**Pull request by priyaayyar**

### CODE REVIEW

File: modules/ROOT/pages/api-gateway-encryption-mule4.adoc [hunk 1]
```diff
@@ -198,8 +198,8 @@ Earlier versions of these policies and policies that are not listed below will n
 | Basic Authentication | 1.2.0 and later.
 | Client ID Enforcement | 1.2.0 and later.
 | Header Injection | 1.1.0 and later.
-| IP Whitelist | 1.2.0 and later.
-| IP Blacklist | 1.2.0 and later.
+| IP Allowlist | 1.2.0 and later.
```
Comment (JaatadiaMulesoft) [inline modules/ROOT/pages/api-gateway-encryption-mule4.adoc]: 1.0.0 and later

File: modules/ROOT/pages/api-gateway-encryption-mule4.adoc [hunk 2]
```diff
@@ -198,8 +198,8 @@ Earlier versions of these policies and policies that are not listed below will n
...
+| IP Blocklist | 1.2.0 and later.
```
Comment (JaatadiaMulesoft) [inline modules/ROOT/pages/api-gateway-encryption-mule4.adoc]: 1.0.0 and later

**Review by JaatadiaMulesoft — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/policy-mule3-custom-policy-references.adoc [hunk 3]
```diff
@@ -143,7 +143,7 @@ Downloading and retrieving client information takes place in the background and
 
 The policy definition file defines the high level properties of the policy, the configurable parameters, labels, and tips that are visible as a UI when implementing the policy.
 
-The following example shows the IP White List policy definition YAML file:
+The following example shows the IP AllowList policy definition YAML file:
```
Comment (tbruno) [inline modules/ROOT/pages/policy-mule3-custom-policy-references.adoc]: Could we align to either use AllowList or Allowlist?

File: modules/ROOT/pages/api-gateway-encryption-mule4.adoc (hunk 1 above)
Comment (priyaayyar) [reply modules/ROOT/pages/api-gateway-encryption-mule4.adoc]: Done! Thanks!

File: modules/ROOT/pages/api-gateway-encryption-mule4.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/pages/api-gateway-encryption-mule4.adoc]: Done! Thanks!

File: modules/ROOT/pages/policy-mule3-custom-policy-references.adoc (hunk 3 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-custom-policy-references.adoc]: Yes, great catch! That was a mistake. The term is "Allowlist". Done! Thanks!

File: modules/ROOT/pages/policy-mule3-ip-blacklist.adoc [hunk 4]
```diff
@@ -45,7 +45,7 @@ You can use a Mule expression to instruct the service to look for the IP address
 
 Insert the IP address in the IP expression field:
 
-image::ip-whitelist-ip-expression.png[ip+blacklist+ip_expression]
+image::ip-allowlist-ip-expression.png[ip+blocklist+ip_expression]
```
Comment (tbruno) [inline modules/ROOT/pages/policy-mule3-ip-blacklist.adoc]: Can you confirm that this change will work Priya?

File: modules/ROOT/pages/policy-mule3-ip-whitelist.adoc [hunk 5]
```diff
@@ -22,41 +22,41 @@ You can set up access in the following ways:
...
 * Define a whole range of IP addresses by stating the relevant octets of the IP address you want to permit. For example, setting `192.168` will include IP addresses from 192.168.0.0 to 192.168.255.255.
 
-To whitelist a list of IP addresses:
+To allowlist a list of IP addresses:
 
-. Enter one or more IP addresses in *Whitelist*.
+. Enter one or more IP addresses in *Allowlist*.
 +
-image::ip-whitelist-051a0.png[height=353,width=377]
+image::ip-allowlist-051a0.png[height=353,width=377]
```
Comment (tbruno) [inline modules/ROOT/pages/policy-mule3-ip-whitelist.adoc:36]: Can you confirm that this change will work Priya?

File: modules/ROOT/pages/policy-mule3-ip-whitelist.adoc [hunk 6]
```diff
@@ -22,41 +22,41 @@ You can set up access in the following ways:
...
 +
 . Click +.
 +
-The greyed-out IP address and a deletion control appears below the Whitelist text entry field.
+The greyed-out IP address and a deletion control appears below the Allowlist text entry field.
 +
-image::ip-whitelist-1b671.png[height=353,width=377]
+image::ip-allowlist-1b671.png[height=353,width=377]
```
Comment (tbruno) [inline modules/ROOT/pages/policy-mule3-ip-whitelist.adoc:42]: Can you confirm that this change will work Priya?

**Review by tbruno — DISMISSED** [review]

**Review by tbruno — DISMISSED** [review]

File: modules/ROOT/pages/policy-mule3-custom-policy-references.adoc [hunk 7]
```diff
@@ -169,15 +169,15 @@ configuration:
     sensitive: false
     allowMultiple: false
   - propertyName: ips
-    name: Whitelist
+    name: Allowlist
```
Comment (JaatadiaMulesoft) [inline modules/ROOT/pages/policy-mule3-custom-policy-references.adoc:172]: we should also modify the line 151 'name' (the 150 'id' should remain as is)

**Review by JaatadiaMulesoft — CHANGES_REQUESTED** [review]

**Review by JaatadiaMulesoft — DISMISSED** [review]

File: modules/ROOT/pages/policy-mule3-ip-whitelist.adoc [hunk 8]
```diff
@@ -22,41 +22,41 @@ You can set up access in the following ways:
...
 * Define a whole range of IP addresses by stating the relevant octets of the IP address you want to permit. For example, setting `192.168` will include IP addresses from 192.168.0.0 to 192.168.255.255.
 
-To whitelist a list of IP addresses:
+To allowlist a list of IP addresses:
 
-. Enter one or more IP addresses in *Whitelist*.
+. Enter one or more IP addresses in *Allowlist*.
 +
-image::ip-whitelist-051a0.png[height=353,width=377]
+image::ip-whitelistlist-051a0.png[height=353,width=377]
```
Comment (sathya0) [inline modules/ROOT/pages/policy-mule3-ip-whitelist.adoc]: Might want to double check this image name

File: modules/ROOT/pages/policy-mule3-reorder-policies-task.adoc [hunk 9]
```diff
@@ -7,12 +7,12 @@ This procedure describes how to re-order policies provided by Anypoint Platform
...
 
-. In Anypoint Platform, click API Manager, and in API Administration, click the API version.
-. On the API version details page, choose Policies.
-. In the top of the list of any applied policies, click Edit Policy Order to change the priority.
-. In Reorder Applied Policies, use the up and down arrows to rearrange the order of policies. +
-For example, if you apply rate limiting, IP whitelist, and XML threat protection, you can reorder only the IP whitelist and XML threat protection policies because rate limiting takes precedence over the other policies.
-. Click Apply order.
+. In Anypoint Platform, click *API Manager*, and in API Administration, click the *API version*.
+. On the API version details page, select *Policies*.
+. At the top of the list of any applied policies, click Edit Policy Order to change the priority.
```
Comment (sathya0) [inline modules/ROOT/pages/policy-mule3-reorder-policies-task.adoc]:
```suggestion
. At the top of the list of any applied policies, click *Edit Policy Order* to change the priority.
```

File: modules/ROOT/pages/policy-mule3-ip-whitelist.adoc (hunk 8 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-ip-whitelist.adoc]: Awesome catch! Thanks Sathya!

File: modules/ROOT/pages/policy-mule3-reorder-policies-task.adoc (hunk 9 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-reorder-policies-task.adoc]: Awesome catch gain! Thanks!

**Review by sathya0 — DISMISSED** [review]

**Review by JaatadiaMulesoft — DISMISSED** [review]

**Review by tbruno — DISMISSED** [review]

**Review by tbruno — APPROVED** [review]

File: modules/ROOT/pages/policy-mule3-ip-blacklist.adoc [hunk 10]
```diff
@@ -45,7 +45,7 @@ You can use a Mule expression to instruct the service to look for the IP address
 
 Insert the IP address in the IP expression field:
 
-image::ip-whitelist-ip-expression.png[ip+blacklist+ip_expression]
+//image::ip-allowlist-ip-expression.png[ip+blocklist+ip_expression]
```
Comment (sathya0) [inline modules/ROOT/pages/policy-mule3-ip-blacklist.adoc:48]: Commented // on purpose?

**Review by sathya0 — APPROVED** [review]

File: modules/ROOT/pages/policy-mule3-ip-blacklist.adoc (hunk 10 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-ip-blacklist.adoc:48]: yes, thanks!

**Pull request merged**